nomos-services/mempool/src/backend/mod.rs

```diff
@@ -50,9 +50,22 @@ pub trait MemPool {
 }
 
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "openapi", derive(utoipa::ToSchema))]
 pub enum Status {
+    /// Unknown status
     Unknown,
+    /// Pending status
     Pending,
+    /// Rejected status
     Rejected,
+    /// Accepted status
+    ///
+    /// The block id of the block that contains the item
+    #[cfg_attr(
+        feature = "openapi",
+        schema(
+            example = "e.g. [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]"
```

Comment:
I think we talked about this, but can this be hex instead?

Reply:
👍